crates/swc_ecma_transforms_module/src/util.rs

```diff
@@ -62,11 +62,20 @@ const fn default_strict_mode() -> bool {
     true
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
+pub struct LazyObjectConfig {
+    pub all_external: bool,
+    pub all_local: bool,
```

Comment:
Sorry, I missed the discord message because I was too busy.

Reply:
I think regex is enough, isn't it?
All local imports starts with './' or '../.

Reply:
Yes, a regex would work. Do you propose adding a new form of config, or reusing `Array<string>` to parse them as regexps? Reusing it could pose problems; I don't know how to differentiate between a regexp and plain strings

Reply:
I think `Lazy::Object(LazyObjectConfig)` where `LazyObjectConfig` has a field like 'pattern' will work

Reply:
Right yeah. A single “pattern” field? I’ll get right on that

Reply:
@kdy1 is this what you had in mind?